examples/samples/timeToLive.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  delay,
+  OnMessage,
+  OnError,
+  MessagingError,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace,
+  SendableMessageInfo,
+  generateUuid
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const queuePath = process.env.QUEUE_NAME || "";
+const deadLetterQueuePath = queuePath + "/$DeadLetterQueue";
```

Comment:
There is a `Namespace.getDeadLetterQueuePathForQueue()` function you can use to get the deadletter queue path

Reply:
Got it, updated this and DLQ samples too.